scripts/support/runner/blackmagick.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2018 Roman Tataurov <diytronic@yandex.ru>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''Runner for flashing with Black Magick Probe.'''
+
+# https://github.com/blacksphere/blackmagic/wiki
+
+import sys
+
+from .core import ZephyrBinaryRunner, RunnerCaps
+
+class BlackMagickRunner(ZephyrBinaryRunner):
+    '''Runner front-end for Black Magick probe.'''
+
+    def __init__(self, gdb, kernel_elf, gdb_port, debug=False):
+        super(BlackMagickRunner, self).__init__(debug=debug)
+        self.gdb = gdb
+        self.kernel_elf = kernel_elf
+        self.gdb_port = gdb_port
+
+    @classmethod
+    def name(cls):
+        return 'blackmagick'
+
+    @classmethod
+    def capabilities(cls):
+        return RunnerCaps(commands={'flash'})
+
+    @classmethod
+    def do_add_parser(cls, parser):
+        parser.add_argument('--gdb_port', default='/dev/ttyACM0',
```

Comment:
Can you rename this to something like --gdb-serial (with dash instead of _)? The --gdb-port argument is used elsewhere to mean a network port.

Reply:
Yes course. Will do.